internal/compiler/fuzz/fuzz_targets/fuzz_compiler.rs

```diff
@@ -0,0 +1,28 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+use i_slint_compiler::diagnostics::{BuildDiagnostics, SourceFile};
+use i_slint_compiler::parser::{parse_tokens, Token};
+
+fuzz_target!(|tokens: Vec<Token>| {
+    let source_file = SourceFile::default();
+    let mut diags = BuildDiagnostics::default();
+
+    let doc_node = parse_tokens(tokens, source_file, &mut diags);
+
+    let rt = tokio::runtime::Runtime::new().unwrap();
+
+    rt.block_on(async move {
```

Comment:
I think that using spin_on here would be a simpler and lighter solution (that also compiles faster) than using tokio.

We use spin_on everywhere else we need to call this Api and where it is fine to block.